SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc

```diff
@@ -37,62 +46,60 @@ ServerTreeView {
 		var collectChildren, levels, countSize;
 		var view, bounds;
 		var pen, font;
+		var drawFunc;
 
 		treeViewStatus.string_(" STARTED: current server tree");
 
 		pen = GUI.current.pen;
-
 		font = Font.sansSerif(10);
 
 		view = UserView.new(viewParent, Rect(0,0,400,400));
 
-		view.drawFunc = {
-			var xtabs = 0, ytabs = 0, drawFunc;
-
-			drawFunc = {|group|
-				var thisSize, rect, endYTabs;
-				xtabs = xtabs + 1;
-				ytabs = ytabs + 1;
-				pen.font = font;
-				group.do({|node|
-					if(node.value.isArray, {
-						thisSize = countSize.value(node);
-						endYTabs = ytabs + thisSize + 0.2;
-						rect = Rect(xtabs * tabSize + 0.5,
-							ytabs * tabSize + 0.5,
-							viewParent.bounds.width - (xtabs * tabSize * 2),
-							thisSize * tabSize;
-						);
-						pen.fillColor = Color.grey(0.8);
-						pen.fillRect(rect);
-						pen.strokeRect(rect);
-						pen.color = Color.black;
-						pen.stringInRect(
-							" Group" + node.key.asString +
-							(node.key == 1).if("- default group", ""),
-							rect
-						);
-						drawFunc.value(node.value);
-						ytabs = endYTabs;
-					},{
-						rect = Rect(xtabs * tabSize + 0.5,
-							ytabs * tabSize + 0.5,
-							7 * tabSize,
-							0.8 * tabSize
-						);
-						pen.fillColor = Color.white;
-						pen.fillRect(rect);
-						pen.strokeRect(rect);
-						pen.color = Color.black;
-						pen.stringInRect(
-							" " ++ node.key.asString + node.value.asString,
-							rect
-						);
-						ytabs = ytabs + 1;
-					});
+		drawFunc = {|group, xtabs = 0, ytabs = 0|
```

Comment:
`xtabs = 0, ytabs = 0` are not used in the call, maybe they can be variables?

Reply:
You are correct. My mistake was due to lack of caution.